doc/Tex/RAM_SCB_viz.tex

```diff
@@ -1,12 +1,5 @@
-RAM-SCB output can be visualized in a number of different ways depending on
-the user's tastes and preferences.  However, a standard library for opening,
-manipulating, and plotting exists as a sub-module of Spacepy.  Users may
-find this freely available Python library at
-\href{http://spacepy.lanl.gov/}{spacepy.lanl.gov}.  Alternatively,
-development versions can be obtained via the project's
-\href{http://sourceforge.net/projects/spacepy/}{Source Forge} page.  The
-RAM-SCB module is nested under {\tt spacepy.pybats.ram}.
+RAM-SCB output can be visualized in a number of different ways depending on the user's tastes and preferences.  However, a standard library for opening, manipulating, and plotting exists as a sub-module of Spacepy.  Users may find this freely available Python library at \href{http://spacepy.lanl.gov/}{spacepy.lanl.gov}.  Alternatively, development versions can be obtained via the project's \href{http://sourceforge.net/projects/spacepy/}{Source Forge} page.  The RAM-SCB module is nested under {\tt spacepy.pybats.ram}.
+% QUESTION: Steve, do we need to update the spacepy location (moved to git didn't it?)? Also, should we expand on this section to show actual examples of the visualization?
```

Comment:
As above, yes to the updated location.

I'd recommend doing the expansion with some simple examples, but maybe as a separate PR once this one is done.

Reply:
K